data/Dockerfiles/letsencrypt/Dockerfile

```diff
@@ -0,0 +1,16 @@
+FROM alpine
+
```

Comment:
Maintainer should be added:
`LABEL maintainer "Andre Peters <andre.peters@servercow.de>"`